pytorch_lightning/utilities/enums.py

```diff
@@ -93,6 +93,8 @@ class DeviceType(LightningEnum):
     >>> # which is case invariant
     >>> DeviceType.TPU in ('tpu', 'CPU')
     True
+    >>> DeviceType.TPU in {'tpu', 'CPU'}
+    True
```

Comment:
what is the diff to the above?
```suggestion
```

Reply:
@Borda It is the test for the change in this PR. You can see it would fail in master

So can you revert the commit?

Reply:
ok, but in such case rather have this in test as it does not give any extra value in examples